resources/packs/core/core.go

```diff
@@ -2,6 +2,10 @@ package core
 
 import "go.mondoo.com/cnquery/resources/packs/core/info"
 
+const MissingUpstreamErr = `To use this resource, you must authenticate with the Mondoo Platform.
+To learn how, read: 
+https://mondoo.com/docs/cnspec/cnspec-adv-install/registration/`
```

Comment:
I find it strange, to see a cnspec URL, when using cnquery. We have the same page also for cnquery: https://mondoo.com/docs/cnquery/cnquery-platform/

Is it possible to show the URL depending on which client is used?

Reply:
I don't know how to do that since this code is directly used into cnspec. Not sure if it's worth it too since the plans are to not require auth for CVEs and advisories too. Might be the case that this err becomes obsolete soon